README.md

```diff
@@ -1,4 +1,79 @@
+
+The output file name format is 'annotated_data_\<timestamp\>.tsv'
+
+## Installation instructions
+
+Change directory to wherever you installed the variant_annotator.py file is and make it executable if it is not already:
+```
+$ chmod 775 variant_annotator.py
+```
+Copy 'variant_annotator.py' to 'variant_annotator' in a bin directory on your path. On my system I chose ~/.local/bin, for example, but if you have a bin directory on your path that you prefer, it will work as well.
+```
+$ cp <path to>/variant_annotator.py ~/.local/bin/variant_annotator
+```
+
+You can then execute the program from the directory of your choice as, correcting for the path of the input file variants.txt:
+```
+$ variant_annotator <variants_txt
+```
+The output file as described above will get written to the directory that the program is called from.
+
+## Output
+### File Output
+The sample here may appear line wrapped, but both the output file and the stdout/stderr have one set of data per line.
+
+The output file name format is 'annotated_data_\<timestamp\>.tsv'
+The file that is produced has tab separated values, one set of annotations per line, and looks like this:
+```
+variant assembly_name   seq_region_name start   end     most_severe_consequence strand  genes
+NC_000001.11:g.215674515G>A     GRCh38  1       215674515       215674515       missense_variant        1       USH2A
+NC_000001.11:g.40819893G>A      GRCh38  1       40819893        40819893        missense_variant        1       KCNQ4
+NC_000002.12:g.39006443C>T      GRCh38  2       39006443        39006443        synonymous_variant      1       SOS1
+NC_000006.12:g.152387156G>A     GRCh38  6       152387156       152387156       synonymous_variant      1       SYNE1
+NC_000001.11:g.215674515G>A     GRCh38  1       215674515       215674515       missense_variant        1       USH2A
+```
+### Screen Output
+The program will display similarly to the file, but with the filename, and will display any variants that errored along with the response code that was returned as it processes the variants:
+```
+The input value: "NC_000001.11:g.40819893T>A" failed with a response of: "400"
+variant assembly_name   seq_region_name start   end     most_severe_consequence strand  genes
+NC_000001.11:g.215674515G>A     GRCh38  1       215674515       215674515       missense_variant        1       USH2A
+NC_000001.11:g.40819893G>A      GRCh38  1       40819893        40819893        missense_variant        1       KCNQ4
+NC_000002.12:g.39006443C>T      GRCh38  2       39006443        39006443        synonymous_variant      1       SOS1
+NC_000006.12:g.152387156G>A     GRCh38  6       152387156       152387156       synonymous_variant      1       SYNE1
+NC_000001.11:g.215674515G>A     GRCh38  1       215674515       215674515       missense_variant        1       USH2A
+
+Done Processing Variants, output written to annotated_data_20231004-134026.tsv
```

Comment:
This shouldn't be in the output, using logging instead of printing would help isolate these messages. As is, this isn't a valid TSV and would likely break a downstream application